src/components/navigation-bar/navigation-bar.component.js

```diff
@@ -26,6 +26,7 @@ const NavigationBar = ({
 NavigationBar.propTypes = {
   /** Styled system spacing props */
   ...propTypes.space,
```

Comment:
We should filter the the prop types here using the `filterStyledSystemPaddingProps` so if a consumer uses any `margin` props they will get the prop type warning. 

https://github.com/Sage/carbon/blob/a99d9679e2a13f789976f3c7bcd02bd411301ca5/src/components/popover-container/popover-container.component.js#L15